components/calendar/calendar.js

```diff
@@ -254,8 +254,8 @@ class Calendar extends LocalizeStaticMixin(LitElement) {
 	firstUpdated(changedProperties) {
 		super.firstUpdated(changedProperties);
 
-		this._today = getToday();
-		const date = this.selectedValue ? parseISODate(this.selectedValue) : this._today;
+		this._today = getDateFromDateObj(getToday());
```

Comment:
I did some reorganizing of the helper functions for consistency in what they're returning, so this and a few other changes in here are a result of that